dat/events/proteron/blockade.lua

```diff
@@ -71,7 +72,8 @@ end
 
 function heartbeat( proteron_blockade )
    local pp = player.pilot()
-   for k,p in ipairs(proteron_blockade) do
+   local stand, _str = faction.get("Proteron"):playerStanding()
+   for k,p in ipairs(proteron_blockade) and stand < 0 do
```

Comment:
I'm guessing you wanted an `if` statement here.

Reply:
I was trying to use the short-circuit fancy stuff in lua